### PR TITLE
FF98 HTMLElement.outerText updates

### PR DESCRIPTION
**Pull request by hamishwillee**
FF98 supports [`HTMLElement.outerText`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/outerText), which was recently added to the spec. This updates the docs to provide a better example of how it is used and note that is its now standard. Also adds release note, and some tidy up of `innerText`

Related docs work tracked in #12576

### CODE REVIEW

**Review by ddbeck — DISMISSED** [review]
A couple little suggestions and a question for you.

(I know you didn't introduce it, but I feel like if we're not confident enough to say rendered without quotes, then we shouldn't be saying it. 😆)

File: files/en-us/web/api/htmlelement/innertext/index.md [hunk 1]
```diff
@@ -11,38 +11,24 @@ browser-compat: api.HTMLElement.innerText
...
+> Basically, `innerText` is aware of the rendered appearance of text, while `textContent` is not.
 
-## Syntax
+## Value
 
-```js
-const renderedText = htmlElement.innerText
-htmlElement.innerText = string
-```
-
```
[the remark this answers is not in the document]
Comment (hamishwillee) [reply files/en-us/web/api/htmlelement/innertext/index.md:33]: Yes, according to https://developer.mozilla.org/en-US/docs/MDN/Structures/Syntax_sections#properties . This was the result of https://github.com/mdn/content/discussions/5992

Note, following having done a few of these, I think explicitly listing as setter and getter _if they differ_ would have been good.

File: files/en-us/web/api/htmlelement/outertext/index.md [hunk 2]
```diff
@@ -5,22 +5,59 @@ tags:
...
+
+Whereas `innerText` changes the text within the selected element. 
+JavaScript like `p.innerText = "Changed!"` changes the content inside the selected `p`.
+The resulting HTML is:
+
+```html
+<div>
+  <p>Changed!</p>
+</div>
+```
```
[the remark this answers is not in the document]
Comment (hamishwillee) [reply files/en-us/web/api/htmlelement/outertext/index.md]: Me either, but it seems silly to try and reinvent the obvious structure. I have added an attribution. Happy to create a new example if you have an idea for a better example.

**Comment by hamishwillee** [comment]
@ddbeck Would appreciate a new look at this please.

File: files/en-us/web/api/htmlelement/innertext/index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (ddbeck) [reply files/en-us/web/api/htmlelement/innertext/index.md:33]: Hmm, yes, but then we'd have to remember to do it, circumstantially. Thanks for mentioning it though (I'm putting in my notes about more rigorously specifying page types).

File: files/en-us/web/api/htmlelement/outertext/index.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (ddbeck) [reply files/en-us/web/api/htmlelement/outertext/index.md]: Yeah, I was hoping I would have a better idea by the time I finished writing my request. I did not. 😆

File: files/en-us/web/api/htmlelement/index.md [hunk 3]
```diff
@@ -45,7 +45,9 @@ _Inherits properties from its parent, {{DOMxRef("Element")}}, and implements tho
 - {{DOMxRef("HTMLElement.inert")}}
   - : A boolean value indicating whether the user agent must act as though the given node is absent for the purposes of user interaction events, in-page text searches ("find in page"), and text selection.
 - {{DOMxRef("HTMLElement.innerText")}}
-  - : Represents the "rendered" text content of a node and its descendants. As a getter, it approximates the text the user would get if they highlighted the contents of the element with the cursor and then copied it to the clipboard.
+  - : Represents the "rendered" text content of a node and its descendants.
```
Comment (ddbeck) [inline files/en-us/web/api/htmlelement/index.md:48]: Ugh, I missed one. I'm going to apply this and merge.

```suggestion
  - : Represents the "rendered" text content of a node and its descendants.
```

**Review by ddbeck — APPROVED** [review]
Looks good. Thank you! 🎉

**Review by ddbeck — COMMENTED** [review]
Oh wait, it won't let me apply the change. Would you pick that up and then we can merge?

**Comment by hamishwillee** [comment]
THanks @ddbeck - merged. FYI only the reason you couldn't merge the suggestion is that you had not modified it to remove the quotation marks - github was telling me that I couldn't merge it because it didn't change the content.

**Comment by ddbeck** [comment]
Oh that's what I get for looking at it at the end of the day. Thanks, Hamish!

**Pull request merged**